vgen/src/main/java/io/gapi/vgen/go/GoLanguageProvider.java

```diff
@@ -50,7 +50,7 @@
 
   @SuppressWarnings("unchecked")
   public <Element> GeneratedResult generate(
-      Element element, SnippetDescriptor snippetDescriptor, GoGapicContext context) {
+      Element element, SnippetDescriptor snippetDescriptor, Object context) {
```

Comment:
@garrettjonesgoogle @vchudnov-g I thought of the same thing, but it would be an empty interface. Are we good with that?